src/libraries/sendtohelix-browser.targets

```diff
@@ -182,7 +175,13 @@
   <Import Project="$(RepositoryEngineeringDir)testing\wasm-provisioning.targets" />
 
   <Target Name="PrepareHelixCorrelationPayload_Wasm">
-    <Error Condition="'$(Scenario)' == '' or '$(Scenario)' == 'normal'" Text="Only supported scenarios are WasmTestOnV8, WasmTestOnNodeJS, WasmTestOnChrome, WasmTestOnFirefox, WasmDebuggerTests and BuildWasmApps at the moment." />
+    <Error Condition="'$(Scenario)' != 'WasmTestOnV8' and 
```

Comment:
Can we refactor to a more common place, so that local run will have the same validation?

Reply:
I can't refactor that because locally empty scenario is valid choice and will get a default. I will add validation of wrong values, bu that would not be shared

Reply:
We can have only the default set locally and later have the same validation